examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1407,13 +1407,19 @@ class GLTFTextureBasisUExtension {
 
 		const textureDef = json.textures[ textureIndex ];
 
+		let extension;
 		if ( ! textureDef.extensions || ! textureDef.extensions[ this.name ] ) {
 
-			return null;
-
+			if (textureDef.source == undefined || json.extensionsRequired != EXTENSIONS.KHR_MATERIALS_UNLIT){
```

Comment:
Note `json.extensionsRequried` is an array, and will never equal `EXTENSIONS.KHR_MATERIALS_UNLIT`.

Reply:
Sorry, the check for extensionsRequired is invalid; I should use array elements instead.